src/bigint.rs

```diff
@@ -125,6 +206,8 @@ pub struct BigUint {
     data: Vec<BigDigit>
 }
 
+type BigUintS<'a> = &'a [BigDigit];
```

Comment:
I don't think this type alias is very helpful, especially since you're still writing out `&mut [BigDigit]` when it needs to be mutable.  Let's leave it all explicit -- it's only a few more characters to write `&[BigDigit]` anyway.